test/tap/umask.js

```diff
@@ -0,0 +1,64 @@
+var path = require("path")
+var test = require("tap").test
+var rimraf = require("rimraf")
+var mkdirp = require("mkdirp")
+var common = require("../common-tap.js")
+var npmlog = require("npmlog")
+var umask = require("../../lib/utils/umask")
+
+var npm = require("../../lib/npm.js")
+
+var pkg = path.resolve(__dirname, "umask")
+
+test("setup", function (t) {
+  rimraf.sync(pkg)
+  mkdirp.sync(pkg)
+  t.end()
+})
+
+test("umask to string", function (t) {
+  t.equal("0000", umask.toString(0))
+  t.equal("0000", umask.toString(0000))
```

Comment:
`eslint --reset` + explicit list of all the rules you want to enable could help with that